x-pack/plugins/elastic_assistant/server/lib/langchain/content_loaders/esql_loader.ts

```diff
@@ -47,15 +49,15 @@ export const loadESQL = async (esStore: ElasticsearchStore, logger: Logger): Pro
       true
     );
 
-    const docs = await docsLoader.load();
-    const languageDocs = await languageLoader.load();
+    const docs = (await docsLoader.load()) as Array<Document<Metadata>>;
+    const languageDocs = (await languageLoader.load()) as Array<Document<Metadata>>;
     const rawExampleQueries = await exampleQueriesLoader.load();
```

Comment:
Will be removing castings over in the KB content PR (as loaders become generalized): https://github.com/elastic/kibana/pull/184885